wolfssl/wolfcrypt/asn_public.h

```diff
@@ -244,7 +244,11 @@ typedef struct WOLFSSL_ASN1_TIME {
 
 enum {
     IV_SZ   = 32,                   /* max iv sz */
+#ifdef WOLFSSL_LONG_PASSWORD
```

Comment:
Should `WOLFSSL_LONG_PASSWORD` get defined automatically for any build cases? Again new macro, which would be nice to avoid. Could we use `OPENSSL_ALL` or something?

Reply:
The long password request came from users of wolfssl-py, and wolfssl-py uses `--enable-opensslall` so guarding with `OPENSSL_ALL` makes sense to me. Changing.

Reply:
Done.